distribution/lib/Standard/Test/0.0.0-dev/src/Test_Reporter.enso

```diff
@@ -278,4 +278,5 @@ type Command_Line_Progress_Reporter
 ## PRIVATE
    Checks if the current process is running in an interactive terminal session.
 is_terminal_interactive -> Boolean =
-    Java_System.console != Nothing
+    console = Java_System.console
+    console != Nothing && console.isTerminal
```

Comment:
this works in NI?

Reply:
You are right, it does not. Pushing the fix in https://github.com/enso-org/enso/pull/12930. But it works on CI, since on CI, it seems that `console != Nothing` evaluates to `False`, and so `console.isTerminal` is not even called.